diesel/src/query_builder/insert_statement.rs

```diff
@@ -351,3 +351,66 @@ impl<'a, T, Table> UndecoratedInsertRecord<Table> for &'a Vec<T> where
     &'a [T]: UndecoratedInsertRecord<Table>,
 {
 }
+
+/// The structure returned by [`insert_default_values`](/diesel/fn.insert_default_values.html). The
+/// only thing that can be done with it is call `into`.
+#[derive(Debug, Copy, Clone, Default)]
+pub struct IncompleteDefaultInsertStatement {}
```

Comment:
I feel like it should be possible to return `IncompleteInsertStatement<DefaultValues, Insert>` where `DefaultValues` implements `Insertable` for all tables rather than introducing a whole new query type for this